kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/templates/kroxylicious/KroxyliciousKafkaClusterRefTemplates.java

```diff
@@ -33,4 +35,20 @@ public static KafkaServiceBuilder defaultKafkaClusterRefDeployment(String namesp
                 .endSpec();
         // @formatter:on
     }
+
+    public static KafkaServiceBuilder kafkaClusterRefDeploymentWithTls(String namespaceName, String clusterRefName) {
+        // @formatter:off
+        return defaultKafkaClusterRefDeployment(namespaceName, clusterRefName)
+                .editSpec()
+                    .withBootstrapServers("%s-kafka-bootstrap.%s.svc.cluster.local:9093".formatted(clusterRefName, Constants.KAFKA_DEFAULT_NAMESPACE))
```

Comment:
It seems odd to read the certificate from the Kafka CR, but not to do the same thing for the `bootstrapServers`.  The bootstrapServers is also available in the listeners array.